.ci/jobs/pull-request-it.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- job:
+    name: "elastic+rally-tracks+pull-request+it"
+    display-name: "elastic / rally-tracks # pull-request+it"
+    description: "Integration tests for rally-tracks pull requests"
+    scm:
+      - git:
+          refspec: "+refs/pull/*:refs/remotes/origin/pr/* +refs/heads/*:refs/remotes/origin/*"
+          branches:
+            - "${ghprbActualCommit}"
+    triggers:
+      - github-pull-request:
+          org-list:
+            - elastic
+          allow-whitelist-orgs-as-admins: true
+          trigger-phrase: '.*run\W+rally-tracks/it.*'
+          github-hooks: true
+          status-context: "rally-tracks/it"
+          cancel-builds-on-update: true
+          black-list-labels:
+            - '>test-mute'
```

Comment:
drive-by comment, I was looking for a way to disable tests for draft PR or PR that I raised and not intended to be merged. This setting will not trigger any PR with the label `>test-mute` so maybe we want to add that label as well.

Reply:
Sure, we can do that.